Earthfile

```diff
@@ -600,7 +600,7 @@ arm-image:
   FROM --platform=linux/arm64 +base-image
   ARG IMAGE_NAME=$(cat /etc/os-release | grep 'KAIROS_ARTIFACT' | sed 's/KAIROS_ARTIFACT=\"//' | sed 's/\"//').img
 
-  FROM --platform=linux/arm64 $OSBUILDER_IMAGE
+  FROM $OSBUILDER_IMAGE
```

Comment:
@Itxaka how come this one has been removed?

Reply:
no worries, I saw you introduced it and then you removed it

Reply:
Yes, sorry today is leftover day for me, I keep leaving stuff around